<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>timeSeriesDialog</class>
 <widget class="QDialog" name="timeSeriesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Time Series Plots</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser" >
   <property name="geometry" >
    <rect>
     <x>7</x>
     <y>344</y>
     <width>545</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>294</y>
     <width>531</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButtonHelp" >
      <property name="text" >
       <string>Help</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSavePlot" >
      <property name="text" >
       <string>Save Plot</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonPlot" >
      <property name="text" >
       <string>Plot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonClose" >
      <property name="text" >
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>13</x>
     <y>12</y>
     <width>531</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>32</x>
      <y>9</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>Select Feature Type:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>30</y>
      <width>471</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab" >
     <attribute name="title" >
      <string>Element Features</string>
     </attribute>
     <widget class="QComboBox" name="comboBoxEleFeature" >
      <property name="geometry" >
       <rect>
        <x>190</x>
        <y>14</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text" >
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Average</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Group IDs</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2" >
      <property name="geometry" >
       <rect>
        <x>22</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text" >
       <string>Select Features by:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxEleVariable" >
      <property name="geometry" >
       <rect>
        <x>190</x>
        <y>85</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text" >
        <string>Interception Storage</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Saturated State</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Unsaturated State</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Surface State</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Evaporation from Canopy</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>ET1</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>ET2</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Precipitation</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Net Precipitation</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Infiltration</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Groundwater Recharge</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3" >
      <property name="geometry" >
       <rect>
        <x>19</x>
        <y>94</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text" >
       <string>Select Plot Variable:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7" >
      <property name="geometry" >
       <rect>
        <x>19</x>
        <y>123</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text" >
       <string>Time-Average Interval</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditEleID" >
      <property name="geometry" >
       <rect>
        <x>191</x>
        <y>50</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip" >
       <string comment="Enter Feature ID" >Enter Feature ID</string>
      </property>
      <property name="text" >
       <string>Enter ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditEleTime" >
      <property name="geometry" >
       <rect>
        <x>190</x>
        <y>120</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip" >
       <string comment="Enter Feature ID" >Enter Time-Average Interval</string>
      </property>
      <property name="text" >
       <string>60</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2" >
     <attribute name="title" >
      <string>River Features</string>
     </attribute>
     <widget class="QLabel" name="label_6" >
      <property name="geometry" >
       <rect>
        <x>22</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text" >
       <string>Select Features by:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxRivFeature" >
      <property name="geometry" >
       <rect>
        <x>190</x>
        <y>14</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text" >
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Average</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Group IDs</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5" >
      <property name="geometry" >
       <rect>
        <x>19</x>
        <y>94</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text" >
       <string>Select Plot Variable:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8" >
      <property name="geometry" >
       <rect>
        <x>19</x>
        <y>123</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text" >
       <string>Time-Average Interval</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditRivID" >
      <property name="geometry" >
       <rect>
        <x>191</x>
        <y>50</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip" >
       <string comment="Enter Feature ID" >Enter Feature ID</string>
      </property>
      <property name="text" >
       <string>Enter Riv ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditRivTime" >
      <property name="geometry" >
       <rect>
        <x>190</x>
        <y>120</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip" >
       <string comment="Enter Feature ID" >Enter Time-Average Interval</string>
      </property>
      <property name="text" >
       <string>60</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxRivVariable" >
      <property name="geometry" >
       <rect>
        <x>190</x>
        <y>85</y>
        <width>191</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text" >
        <string>Head</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Inflow</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Outflow</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Base Flow</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Surface Flow</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Base Flow Left</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Base Flow Right</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Surface Flow Left</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Surface Flow Right</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEditFileName" >
    <property name="geometry" >
     <rect>
      <x>167</x>
      <y>234</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>37</x>
      <y>236</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text" >
     <string>Model Output File:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2" >
    <property name="geometry" >
     <rect>
      <x>416</x>
      <y>225</y>
      <width>77</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonBrowse" >
       <property name="text" >
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>timeSeriesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
